### PR TITLE
Fix Airlock blocked functionality

### PR DESCRIPTION
**Pull request by anatbal**
# PR for issue [#1954](https://github.com/microsoft/AzureTRE/issues/1954)

## What is being addressed

Copying to blocked container was not happening due to some bugs. This PR has the fixes

### CODE REVIEW

File: airlock_processor/ScanResultTrigger/__init__.py [hunk 1]
```diff
@@ -17,13 +18,13 @@ def main(msg: func.ServiceBusMessage,
...
     try:
-        enable_malware_scanning = os.environ["ENABLE_MALWARE_SCANNING"]
+        enable_malware_scanning = strtobool(os.environ["ENABLE_MALWARE_SCANNING"])
     except KeyError as e:
         logging.error("environment variable 'ENABLE_MALWARE_SCANNING' does not exists. cannot continue.")
         raise e
 
     # Sanity
-    if enable_malware_scanning is False:
+    if enable_malware_scanning:
```
Comment (eladiw) [inline airlock_processor/ScanResultTrigger/__init__.py:27]: incorrect rule. should be ==false

Comment (anatbal) [reply airlock_processor/ScanResultTrigger/__init__.py]: fixed

**Review by eladiw — APPROVED** [review]

**Pull request merged**